### PR TITLE
Sylvia/docs/doc 1770

### PR DESCRIPTION
**Pull request by srpearce**
Adding info about Tags view (Student Notes tool/Notes page in courseware).

### CODE REVIEW

File: en_us/students/source/SFD_student_notes.rst [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+  note remains open until you click outside the highlighted text.
+
+* You can view all the notes you've made in the course on the **Notes** page. You
+  can also search your notes on this page.
+
+.. _The Notes Page:
+
+================
+The Notes Page
+================
```
Comment (srpearce) [inline en_us/students/source/SFD_student_notes.rst]: @lamagnifica @cahrens This section ("The Notes Page") has been updated to include information about tags; please review it and let me know your thoughts. You can see a built version at http://draft-sn-tags.readthedocs.org/en/latest/SFD_student_notes.html?#the-notes-page.

Comment (cahrens) [inline en_us/students/source/SFD_student_notes.rst]: Looks good to me!

File: en_us/shared/exercises_tools/student_notes.rst [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+This tool is enabled by default. You do not have to do anything before
+students can use this tool.
+
+Students can access their notes either in the body of the course or on a
+**Notes** page. On the **Notes** page, a student can see all the notes that the
+student has made during the course. The page also contains links to the
+location of the notes in the course content.
+
+.. image:: ../../../shared/building_and_running_chapters/Images/SFD_SN_NotesPage.png
+  :width: 500
```
Comment (lamagnifica) [inline en_us/shared/exercises_tools/student_notes.rst]: In this image, erase the "Instructor" option

File: en_us/shared/exercises_tools/create_exercises_and_tools.rst [hunk 3]
```diff
@@ -146,6 +146,13 @@ General Exercises and Tools
      - If you have an problem that is already written in LaTeX, you can use
        this problem type to easily convert your code into XML.
      - No support
+   * - .. image:: ../../../shared/building_and_running_chapters/Images/SFD_SN_bodyexample.png
+          :width: 100
+          :alt: Image of student notes in a course
+     - :ref:`Student Notes Tool`
+     - With student notes, students can make notes about what they read and
```
Comment (lamagnifica) [inline en_us/shared/exercises_tools/create_exercises_and_tools.rst]: change "students" to "learners"

File: en_us/shared/exercises_tools/student_notes.rst [hunk 4]
```diff
@@ -0,0 +1,34 @@
+.. _Student Notes Tool:
+
+##############################
+Student Notes Tool
+##############################
+
+The Student Notes tool allows students to highlight and make notes about what
```
Comment (lamagnifica) [inline en_us/shared/exercises_tools/student_notes.rst]: students --> learners

File: en_us/shared/exercises_tools/student_notes.rst [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+they read in the body of the course.
```
Comment (lamagnifica) [inline en_us/shared/exercises_tools/student_notes.rst]: in the body of the course --> in the courseware? in course HTML components? (suggest rewording to be more consistent or more specific)

File: en_us/shared/exercises_tools/student_notes.rst [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+  :width: 500
+  :alt: Image of a course page that includes highlighted text and a student
+      note.
+
+.. note:: The Student Notes tool is available for text, including text in 
+ HTML components. However, the tool is currently not available for discussions,
+ exercises, video transcripts, or PDF documents.
+
+This tool is enabled by default. You do not have to do anything before
+students can use this tool.
```
Comment (lamagnifica) [inline en_us/shared/exercises_tools/student_notes.rst]: learners (throughout), except name of tool

File: en_us/shared/exercises_tools/student_notes.rst [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+
+Students can access their notes either in the body of the course or on a
```
Comment (lamagnifica) [inline en_us/shared/exercises_tools/student_notes.rst]: in the body --> same comment

File: en_us/shared/exercises_tools/student_notes.rst [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+**Notes** page. On the **Notes** page, a student can see all the notes that the
+student has made during the course. The page also contains links to the
```
Comment (lamagnifica) [inline en_us/shared/exercises_tools/student_notes.rst]: the student --> she (or he)

File: en_us/shared/exercises_tools/student_notes.rst [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+location of the notes in the course content.
+
+.. image:: ../../../shared/building_and_running_chapters/Images/SFD_SN_NotesPage.png
+  :width: 500
+  :alt: The Student Notes page listing all the notes a student has made in the
+      course.
+
+For detailed information about the ways students can use student notes, see
+`Student Notes <http://edx-guide-for-
```
Comment (lamagnifica) [inline en_us/shared/exercises_tools/student_notes.rst]: the ways ^that^ students
in a previous round of edits I believe @explorerleslie agreed that when presenting info about this tool to learners, we can just call it "notes" and drop "student"

File: en_us/students/source/SFD_student_notes.rst [hunk 10]
```diff
@@ -0,0 +1,270 @@
...
+Show or Hide Notes
+***************************
+
+By default, all notes that you make are visible. However, you can show or hide
+notes by selecting the the pencil icon in the lower right corner. When the
+pencil icon has a dark gray background, notes are visible. When the pencil
+icon has a light gray background, notes are hidden.
+
+.. image:: /Images/SFD_SN_NotesShowHide.png
+  :width: 400
```
Comment (lamagnifica) [inline en_us/students/source/SFD_student_notes.rst]: was the text bold already, and then highlighted? in the example, it's hard to tell if the act of selecting the text to make a note both bolded it and highlighted it, or if it was already bolded.

File: en_us/students/source/SFD_student_notes.rst [hunk 11]
```diff
@@ -0,0 +1,270 @@
+.. _SFD Student Notes:
+
+##############################
+Student Notes
+##############################
```
Comment (lamagnifica) [inline en_us/students/source/SFD_student_notes.rst]: as per previous discussion, just call this feature "notes" for our learner audience. Heading should start with a gerund: "Taking Notes", "Making Notes", etc.

File: en_us/students/source/SFD_student_notes.rst [hunk 12]
```diff
@@ -0,0 +1,270 @@
...
+
+As you work through an edX course, you may want to highlight a particular
```
Comment (lamagnifica) [inline en_us/students/source/SFD_student_notes.rst]: may --> might

File: en_us/students/source/SFD_student_notes.rst [hunk 13]
```diff
@@ -0,0 +1,270 @@
...
+passage or make a note about what you have read. In some edX courses, you can
+highlight passages and make notes right in the body of the course. Your notes
+can contain both text and tags to help you organize your notes.
+
+.. note:: You can create notes for most text in the body of the course. 
+ However, notes are currently not available for exercises, videos, or PDF
+ textbooks.
+
+For more information about working with notes, see the following sections.
```
Comment (lamagnifica) [inline en_us/students/source/SFD_student_notes.rst]: replace lines 16- 23 with

`.. contents:: Section Contents 
   :local:
   :depth: 1`

(with the line feeds, without the graves)

File: en_us/students/source/SFD_student_notes.rst [hunk 14]
```diff
@@ -0,0 +1,270 @@
...
+
+* :ref:`SFD SN Introduction`
+* :ref:`SFD SN Add Note`
+* :ref:`SFD SN View Note`
+* :ref:`SFD SN Edit Note`
+* :ref:`SFD SN Show Notes`
+* :ref:`SFD SN Search Notes`
+
+.. note:: You can use either the mouse or keyboard shortcuts to create,
```
Comment (lamagnifica) [inline en_us/students/source/SFD_student_notes.rst]: the --> your

File: en_us/students/source/SFD_student_notes.rst [hunk 15]
```diff
@@ -0,0 +1,270 @@
...
+ access, and delete notes. For more information about using keyboard
+ shortcuts, see :ref:`SFD SN Keyboard Shortcuts`.
+
+.. _SFD SN Introduction:
+
+***************************
+Introduction
+*************************** 
+
+When a course includes notes, every page has a **Notes** page at the
```
Comment (lamagnifica) [inline en_us/students/source/SFD_student_notes.rst]: "includes notes" meaning "notes that you have already taken", or "offers the notes feature"? not entirely clear right now.
When a course includes the notes feature,

File: en_us/students/source/SFD_student_notes.rst [hunk 16]
```diff
@@ -0,0 +1,270 @@
...
+top and a pencil icon in the lower right corner. 
+
+.. image:: /Images/SFD_SN_notesindicators.png
+  :width: 500
+  :alt: Page of course content that includes the Notes page at the top,
+      highlighted text with an accompanying note, and the pencil icon in the
+      lower right corner.
+
+You can see individual notes inside the course content, or you can see all of
```
Comment (lamagnifica) [inline en_us/students/source/SFD_student_notes.rst]: After you begin to make notes, you can...

File: en_us/students/source/SFD_student_notes.rst [hunk 17]
```diff
@@ -0,0 +1,270 @@
...
+Notes Page`.
+
+
+.. _SFD SN Add Note:
+
+******************************************
+Highlight a Passage or Add a Note
+******************************************
+
+To highlight a passage or add a note that includes text and tags on a page,
```
Comment (lamagnifica) [inline en_us/students/source/SFD_student_notes.rst]: strike "on a page" - awkward, not needed

File: en_us/students/source/SFD_student_notes.rst [hunk 18]
```diff
@@ -0,0 +1,270 @@
...
+follow these steps.
+
+#. Select the text that you want to highlight or make the note about. You can
+   select as much text as you want.
+
+#. When a pencil "edit" icon appears above the selected text, click
```
Comment (lamagnifica) [inline en_us/students/source/SFD_student_notes.rst]: click --> select (throughout)

File: en_us/students/source/SFD_student_notes.rst [hunk 19]
```diff
@@ -0,0 +1,270 @@
...
+     :width: 400
+     :alt: Paragraph with text selected and the pencil icon above the selected
+         text.
+
+#. When the note editor opens, enter your note and any tags that you want to
+   add. You can also save the highlight for the passage without entering a
+   note or tag.
+
+   * To highlight a passage without adding a note or tag, select **Save** or
+     press Enter. When you hover over the highlighted text, the note field
```
Comment (lamagnifica) [inline en_us/students/source/SFD_student_notes.rst]: press Enter ^immediately. ?
When you hover --> When you move your cursor

File: en_us/students/source/SFD_student_notes.rst [hunk 20]
```diff
@@ -0,0 +1,270 @@
...
+.. image:: /Images/SFD_SN_notesindicators.png
+  :width: 500
+  :alt: Page of course content that includes the Notes page at the top,
+      highlighted text with an accompanying note, and the pencil icon in the
+      lower right corner.
+
+You can see individual notes inside the course content, or you can see all of
+your notes at once on the **Notes** page. For more information, see :ref:`The
+Notes Page`.
+
```
Comment (lamagnifica) [inline en_us/students/source/SFD_student_notes.rst]: maybe introduce the tag concept up here by saying something like "When you make notes, you can include optional tags. Adding tags to your notes can help you organize them into different categories, making followup easier later." Image might even include a tag...

File: en_us/students/source/SFD_student_notes.rst [hunk 21]
```diff
@@ -0,0 +1,270 @@
...
+     Tags cannot contain spaces. If you want to add a tag that has more than
+     one word, type multiple words as one word with no spaces, or use hyphens
+     (-) or underscores (_) to separate words in the tag.
+
+     .. image:: /Images/SFD_SN_NoteEditor.png
+      :width: 400
+      :alt: Note editor open with a learner's note.
+
+3. After you finish writing your note and adding tags, select **Save** or
+   press Enter. The note editor closes, and the text that you selected remains
```
Comment (lamagnifica) [inline en_us/students/source/SFD_student_notes.rst]: strike "After you finish writing your note and adding tags, "

File: en_us/students/source/SFD_student_notes.rst [hunk 22]
```diff
@@ -0,0 +1,270 @@
...
+   highlighted.
+
+.. _SFD SN View Note:
+
+***************************
+View Notes
```
Comment (lamagnifica) [inline en_us/students/source/SFD_student_notes.rst]: gerund

File: en_us/students/source/SFD_student_notes.rst [hunk 23]
```diff
@@ -0,0 +1,270 @@
...
+  over or select the highlighted text. When you select the highlighted text, your
+  note remains open until you click outside the highlighted text.
+
+* You can view all the notes you've made in the course on the **Notes** page. You
+  can also search your notes on this page.
+
+.. _The Notes Page:
+
+================
+The Notes Page
```
Comment (lamagnifica) [inline en_us/students/source/SFD_student_notes.rst]: infinitive

File: en_us/students/source/SFD_student_notes.rst [hunk 24]
```diff
@@ -0,0 +1,270 @@
...
+
+.. _SFD SN View Note:
+
+***************************
+View Notes
+***************************
+
+You can view your course notes in two places. 
+
+* You can view individual notes in the place in the course where you created the
```
Comment (lamagnifica) [inline en_us/students/source/SFD_student_notes.rst]: "in the place in the course" --> "on the page"

File: en_us/students/source/SFD_student_notes.rst [hunk 25]
```diff
@@ -0,0 +1,270 @@
...
+  notes. To see your notes, go to the unit that contains your note and hover
+  over or select the highlighted text. When you select the highlighted text, your
```
Comment (lamagnifica) [inline en_us/students/source/SFD_student_notes.rst]: "hover over" --> "move the cursor over" or just strike this alternative

File: en_us/students/source/SFD_student_notes.rst [hunk 26]
```diff
@@ -0,0 +1,270 @@
...
+  note remains open until you click outside the highlighted text.
+
+* You can view all the notes you've made in the course on the **Notes** page. You
```
Comment (lamagnifica) [inline en_us/students/source/SFD_student_notes.rst]: you've --> that you have (throughout, check for contractions)

File: en_us/students/source/SFD_student_notes.rst [hunk 27]
```diff
@@ -0,0 +1,270 @@
...
+* When you select **Recent Activity**, notes are sorted by the date you
+  created or edited them, with the most recently modified first.
+
+* When you select **Location in Course**, notes are sorted sequentially by the
+  location of the note in the courseware. 
+
+* When you select **Tags**, notes are sorted by the tags you have added.
+
+  * The most frequently used tags are at the top of the list. The number of
+    notes that have that tag appears in parentheses after the tag.
```
Comment (lamagnifica) [inline en_us/students/source/SFD_student_notes.rst]: that tag --> a tag

File: en_us/students/source/SFD_student_notes.rst [hunk 28]
```diff
@@ -0,0 +1,270 @@
...
+
+.. _SFD SN Edit Note:
+
+************************************
+Edit or Delete a Note or Highlight
+************************************
+
+To edit a note, follow these steps.
+
+#. In the body of the course, hover over the highlighted text until your note
```
Comment (lamagnifica) [inline en_us/students/source/SFD_student_notes.rst]: hover - same edit (I cannot levitate! don't make me feel bad about myself!)
throughout

File: en_us/students/source/SFD_student_notes.rst [hunk 29]
```diff
@@ -0,0 +1,270 @@
...
+   appears.
+#. When the note appears, select the pencil icon in the upper right corner to
+   open the note editor.
+
+   .. image:: /Images/SFD_SN_EditDeleteNote.png
+    :width: 175
+    :alt: Note editor with the pencil and paper and X icons visible.
+
+#. In the note editor, edit your note, and then click **Save**.
+
```
Comment (lamagnifica) [inline en_us/students/source/SFD_student_notes.rst]: If you have time, try a little exercise to reduce the word count. For example, in this procedure would anything be lost if you began step 2 without "When the note appears"?, or step 3 without "In the note editor"?

File: en_us/students/source/SFD_student_notes.rst [hunk 30]
```diff
@@ -0,0 +1,270 @@
...
+#. When the note appears, select the trash icon in the upper right corner to
+   delete the note or highlight.
+
+.. _SFD SN Show Notes:
+
+***************************
+Show or Hide Notes
+***************************
+
+By default, all notes that you make are visible. However, you can show or hide
```
Comment (lamagnifica) [inline en_us/students/source/SFD_student_notes.rst]: By default, you can see all of your notes. You can hide your notes, and show them again, by selecting...

File: en_us/students/source/SFD_student_notes.rst [hunk 31]
```diff
@@ -0,0 +1,270 @@
...
+#. At the top of any courseware page, select **Notes** to open the **Notes**
+   page.
+#. In the upper right corner of the page, type a search term in the **Search
+   notes for** field.
+#. Click the magnifying glass icon or press Enter.
+
+.. _SFD SN Keyboard Shortcuts:
+
+*********************************************
+Keyboard Shortcuts for Student Notes
```
Comment (lamagnifica) [inline en_us/students/source/SFD_student_notes.rst]: Using...

**Comment by lamagnifica** [comment]
a couple of suggestions, mostly style guide copy editing...

File: en_us/shared/exercises_tools/student_notes.rst (hunk 9 above)
Comment (explorerleslie) [inline en_us/shared/exercises_tools/student_notes.rst]: yes, we can drop "student"

**Comment by srpearce** [comment]
Superseded by [PR 377](https://github.com/edx/edx-documentation/pull/346)
